### DISCUSSION TITLE
Can I use Kind 2 Java API in Maven to build just a normal lustre program as a controller ?

### DISCUSSION THREAD

**Discussion by Atmanuclear** [kind2-mc/kind2#949] [Q&A] [answered]
I found that Kind 2 Java API is provided in Maven. My interest is to write a lustre controller program to satisfy some requirement, so I want to first use some API  to build my lustre program, I wonder if this API can work well ?

**Comment by daniel-larraz** [comment, marked as answer]
Yes, you can use [Kind 2 Java API](https://github.com/kind2-mc/kind2-java-api) to build a Lustre program. You can find the steps to do that [here](https://github.com/kind2-mc/kind2-java-api#api-usage).

**Reply by Atmanuclear** [reply]
Thank you very much for your prompt reply, which give me a big surprise ! I will carefully read the link and try it out. Thanks again!
